# r2params.yaml
# Contains PD control gains for Raven.
#   Warning: the order of the gains is very importannt.
#     Of particular note, the order is different for R_I vs R_II.221
#     Position axes are 1-3.  4th is not used.  5-8 are tool axes.
#
gains_gold_kp:
- 0.060
- 0.050
- 0.020
- 0.00
- 0.09
- 0.05
- 0.02
- 0.02
gains_green_kp:
- 0.060
- 0.050
- 0.020
- 0.00
- 0.09
- 0.05
- 0.02
- 0.02
gains_gold_kd:
- 0.003
- 0.004
- 0.001
- 0.00
- 0.001 # was 0.001
- 0.00
- 0.00
- 0.00
gains_green_kd:
- 0.003
- 0.004
- 0.001
- 0.00
- 0.001 # was 0.001
- 0.00
- 0.00
- 0.00

gains_gold_ki:
- 0.00
- 0.00
- 0.00
- 0.00
- 0.05
- 0.05
- 0.02
- 0.02
gains_green_ki:
- 0.00
- 0.00
- 0.00
- 0.00
- 0.05
- 0.05
- 0.02
- 0.02

max_position:
  shoulder:
    gold: 0
    green: 5
  elbow:
    gold: 0
    green: 6.75
  insertion:
    gold: 0.064
  rotation:
    gold: 160
    green: -35
  wrist:
    green: 0
  grasp:
    green: -17

ik_INACTIVE:
  max_diff:
    shoulder: 10
    elbow: 10
    insertion: 0.005
    rotation: 10
    wrist: 10
    grasp: 100
